.github/workflows/ci.yml

```diff
@@ -27,7 +27,7 @@ jobs:
       if: github.ref != 'refs/heads/main'
       run: npm run build
       env:
-        SHOW_NETLIFY_BADGE: true
+        VITE_SHOW_NETLIFY_BADGE: true
```

Comment:
This will build with netlify, but the deploy_preview code below is still commented out. Is this something you might be able to debug and fix?

Reply:
I don't have access to netlify so it's hard to debug. Preview deploys are working in my Netlify account without that. What's the purpose of this job?

Reply:
I'm guessing it was to preview each build using the account associated with the github

Reply:
Is it different from the built-in feature in Netlify? I'm not familiar enough